.circleci/config.yml

```diff
@@ -205,8 +205,8 @@ jobs:
       - run:
           name: Upload contract data
           command: |
-            cd /tmp/keep-tecdsa/contracts
-            gsutil -m cp * gs://${CONTRACT_DATA_BUCKET}/keep-tecdsa
+            cd /tmp/keep-ecdsa/contracts
+            gsutil -m cp * gs://${CONTRACT_DATA_BUCKET}/keep-ecdsa
```

Comment:
Do we need to update any other repos using this bucket? `tbtc` for instance?

Reply:
Good catch: https://github.com/keep-network/tbtc/blob/b87758e91960b0f2446e5355ef422a8bfcc46910/implementation/scripts/circleci-provision-external-contracts.sh#L10

This should go away once packaging is operating properly…